docs/container_registry.md

```diff
@@ -136,6 +136,8 @@ services:
   registry:
     restart: always
     image: registry:2.4.1
+    ports:
+    - "5000:5000"
```

Comment:
This is not required. And gitlab export the port and proxy the access to the registry.